server/http_handler.go

```diff
@@ -718,6 +718,17 @@ func (h settingsHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 			config.StoreGlobalConfig(cfg)
 			deadlockhistory.GlobalDeadlockHistory.Resize(uint(capacity))
 		}
+		if lockUniqueKey := req.Form.Get("tidb_lock_unique_key"); lockUniqueKey != "" {
+			switch lockUniqueKey {
+			case "0":
+				variable.LockUniqueKeys.Store(false)
+			case "1":
+				variable.LockUniqueKeys.Store(true)
+			default:
+				writeError(w, errors.New("illegal argument, please use 0 or 1 for this"))
+				return
+			}
+		}
 	} else {
 		writeData(w, config.GetGlobalConfig())
```

Comment:
maybe we need a way to get the current configuration value..

for example: someone reports a oncall issue and we want to confirm is it running in false or true status~?

or we can write some log to indicate some changed it

Reply:
`select @@tidb_lock_unique_key` won't work?

Reply:
Logs are added changing the config, it may need more work to track the status infomation in the tidb log file.

Reply:
I see, tidb_lock_unique_key is not a variable.

Reply:
@jackysp 
It's a http api
`curl -X POST -d "tidb_lock_unique_key=1" http://127.0.0.1:10080/settings`
As it's a temporary solution, better not to add system variables or configurations.

Reply:
So it is not necessary to update the docs of http api?

Reply:
Not quite sure, as it's temporary solution better not to make it very public. Also it's expected to be optimized or solved in the next release if possible.